test/terraform_module_test.go

```diff
@@ -52,13 +89,22 @@ func TestExamplesForTerraformModules(t *testing.T) {
 		},
 	}
 
+	for _, tests := range tests {
+		runTest(t, tests)
+	}
+}
+
+func runTest(t *testing.T, tests []TestCase) {
 	// Run tests in parallel
-	for _, tt := range tests {
+	for i, tt := range tests {
 		workingDir := tt.workingDir
 		genTestDataFunc := tt.genTestDataFunc
 		validateFunc := tt.validateFunc
 		t.Run(tt.name, func(t *testing.T) {
-			t.Parallel()
+			// If not the last test, run in parallel
+			if i != len(tests)-1 {
```

Comment:
How come only the last test we are not running in parallel?

Reply:
So the idea is that the tests are grouped and we will run all the tests in the group in pararrell except the last one so that the last one has to finish before we can move onto the next group.